public/js/createCharacter.js

```diff
@@ -40,6 +40,6 @@ function createChar(name, attack, defense, health) {
         defense: defense,
         health: health
     }).then(() => {
-        window.location.href = "/game"
+        window.location.href = "/selecthero"
```

Comment:
this page ( createCharacter.js ) is the original JS for the createCharacter.html page that you have already deleted
this page is not linked to any handlebars or html page, (we can delete it)